samigo/samigo-app/src/webapp/jsf/author/publishedSettings.jsp

```diff
@@ -245,6 +250,10 @@
                         styleClass="form-control-label col-md-2"/>
         <div class=" col-md-10">
             <h:outputText value="#{publishedSettings.publishedUrl}" />
+            <button type="button" id="copyToClipboard" title="<h:outputText value='#{assessmentSettingsMessages.copyToClipboard}' />">
+                <span class="fa fa-clipboard" aria-hidden="true"></span>
+                <span class="hide"><h:outputText value="#{assessmentSettingsMessages.copyToClipboard}" /></span>
```

Comment:
Is this meant to be hidden from everyone except screen readers? If yes, please just use sr-only instead

Reply:
Yes I was just in the process of changing that.